build.sbt

```diff
@@ -20,7 +20,7 @@ lazy val migration = project
   .disablePlugins(MimaPlugin)
   .settings(
     name := "akka-persistence-postgres-migration",
-    libraryDependencies ++= Dependencies.Migration,
+    libraryDependencies ++= Dependencies.Libraries ++ Dependencies.Migration,
```

Comment:
This change isn't needed as `migration` depends on `core` already

Reply:
yes it would seem so but the tests refuse to compile without this explicit addition, didn't dig further as to why that was the case

Reply:
@jchapuis indeed, this change is not required. 

On the `migration` dependencies you added the `akka-persistence-query` in the Seq that ios being mapped to the `Test` scope. If you make it available on `Compile`, this change is not required.

Reply:
aha good catch sorry I had not spotted the scope mapping there. Changed it